src/test/java/wedlog/address/logic/parser/VendorCommandParserTest.java

```diff
@@ -23,12 +24,9 @@ public class VendorCommandParserTest {
     @Test
     public void parseCommand_vendorAdd() throws Exception {
         String input = "add n/vendor p/102391";
-        // expected type, expected string, executable
-        assertThrows(ParseException.class, VENDOR_ADD_PARSE_EXCEPTION_MSG, () -> parser.parseCommand(input));
```

Comment:
Perhaps `VENDOR_ADD_PARSE_EXCEPTION_MSG` can be removed from the class?

Reply:
Sorry, what is the rationale behind this?